packages/rollup-core/src/app/constants.ts

```diff
@@ -44,3 +44,32 @@ export const L2_TO_L1_MESSAGE_PASSER_OVM_ADDRESS =
 
 // See the getTransactionBatchCalldata(...) function of canonical-chain-batch-submitter.ts for more info
 export const ROLLUP_TX_SIZE_IN_BYTES_MINUS_CALLDATA = 150
+
+/*
+Static bytes: 160
+- 4 (Method ID)
+- 32 (txs byte start)
+- 32 (timestamp)
+- 32 (block number)
+- 32 (starts at index)
+- 32 (number of txs bytes elements)
```

Comment:
@ben-chain you might be good at double checking to make sure there's nothing missing in these calculations. I don't spot any issues off the top of my head though